exporter/kafkaexporter/README.md

```diff
@@ -2,16 +2,20 @@
 
 Kafka exporter exports traces to Kafka. This exporter uses a synchronous producer
 that blocks and does not batch messages, therefore it should be used with batch and queued retry
-processors for higher throughput and resiliency.
+processors for higher throughput and resiliency. Message payload encoding is configurable.
  
-Message payloads are serialized OTLP `ExportTraceServiceRequest`.
-
 The following settings are required:
 - `protocol_version` (no default): Kafka protocol version e.g. 2.0.0
 
 The following settings can be optionally configured:
 - `brokers` (default = localhost:9092): The list of kafka brokers
 - `topic` (default = otlp_spans): The name of the kafka topic to export to
+- `encoding` (default = otlp_proto): The encoding of the payload sent to kafka. Available encodings:
+  - `otlp_proto`: the payload is serialized to `ExportTraceServiceRequest`.
+  - `jaeger_proto`: the payload is serialized to a single Jaeger proto `Span`.
+  - `jaeger_json`: the payload is serialized to a single Jaeger JSON Span using `jsonpb`.
+  - `zipkin_thrift`: the payload is serialized into Zipkin Thrift spans.
```

Comment:
I would not support this, better to support zipkin_proto. Would like to avoid thrift dependencies in core if possible.

Reply:
Zipkin does not support proto encoding in Kafka https://github.com/openzipkin/zipkin/tree/master/zipkin-collector/kafka#encoding-spans-into-kafka-messages.

Supporting thrift and JSON allows us to help migrate users from Jaeger to other systems. The Zipkin thrift is already supported by Zipkin receiver. Are there plans to drop it?

Reply:
Maybe I am wrong but I did not see thrift in the receiver.

Reply:
The receiver handles Zipkin thrift spans https://github.com/open-telemetry/opentelemetry-collector/blob/master/receiver/zipkinreceiver/trace_receiver.go#L120